Sources/UIWindow.swift

```diff
@@ -57,6 +65,16 @@ public class UIWindow: UIView {
     }
 }
 
+private extension UITouch {
+    var containsTapRecognizersWhichCancelsTouchesInHitview: Bool {
+        return gestureRecognizers.contains(where: { $0 is UITapGestureRecognizer && $0.state == .began && $0.cancelsTouchesInView })
+    }
+
+    var containsRecognizersWhichCancelsTouchesInHitview: Bool {
+        return gestureRecognizers.contains(where: { ($0.state == .began || $0.state == .changed) && $0.cancelsTouchesInView })
+    }
+}
```

Comment:
we are now checking for .began and .changed, which is correct I think

Reply:
because we want tapGestureRecognizers to cancel touchesBegan on hitview (and the hitview responder)